manim/mobject/boolean_ops.py

```diff
@@ -0,0 +1,291 @@
+import typing
```

Comment:
> how it's done in the rest of the library?

I could see it is done both ways. Is there any standard that was decided on?

Reply:
Hmm, not that I know of. Should we do a poll on Discord?

Reply:
While I agree with the poll on discord, I think adding typing in front of everything is a bit cumbersome. When you refer to a type, it's quite clear that it's a type.

Reply:
For me I prefer importing `typing` module or aliased it as something short say `T`, `import typing as T`. I prefer so because remembering to add all the import beforehand seems a lot of work, rather using as module seems better.

Reply:
For now, let's have this. Once we set a standard we can change this.

Reply:
Alright, fair enough.